src/modules/household/components/householdActions/SuccessWayfindingStep.tsx

```diff
@@ -0,0 +1,68 @@
+import { Alert, AlertTitle, Button, Stack } from '@mui/material';
+import React from 'react';
+import { generatePath } from 'react-router-dom';
+import ButtonLink from '@/components/elements/ButtonLink';
+import { clientBriefName } from '@/modules/hmis/hmisUtil';
+import {
+  EnrollmentDashboardRoutes,
+  ProjectDashboardRoutes,
+} from '@/routes/routes';
+import {
+  HouseholdClientFieldsFragment,
+  ProjectAllFieldsFragment,
+} from '@/types/gqlTypes';
+
+interface Props {
+  title: string;
+  description: string;
+  primaryClientName: string;
+  secondary?: HouseholdClientFieldsFragment;
+  project: Pick<ProjectAllFieldsFragment, 'id' | 'projectName'>;
+  onClose: VoidFunction;
+}
+
+const SuccessWayfindingStep = ({
```

Comment:
We may want to pull this out into a generic wayfinding component at some point, but we can wait until next time we add a wayfinding step.